testing/run_tests.py

```diff
@@ -837,7 +837,8 @@ def GatherConstFinderTests(build_dir):
       '--disable-dart-dev',
       os.path.join(test_dir, 'const_finder_test.dart'),
       os.path.join(build_dir, 'gen', 'frontend_server.dart.snapshot'),
-      os.path.join(build_dir, 'flutter_patched_sdk')
+      os.path.join(build_dir, 'flutter_patched_sdk'),
+      os.path.join(build_dir, 'dart-sdk', 'lib', 'libraries.json')
```

Comment:
This path is needed to do web compilation

Reply:
Maybe we should pass a Flutter-specific `flutter/web_sdk/libraries.json` file instead.

Reply:
you're right, I double checked what we do in the tool, and it's actually that file, will update

Reply:
Actually, using that file does not work in this test as for dart2js is just points to a relative path to the dart-sdk libraries.json https://github.com/flutter/engine/blob/main/web_sdk/libraries.json#L26. This path (`"../dart-sdk/lib/libraries.json"`) is relative to its position in the Flutter cache, however, not the engine workspace.

Passing the dart-sdk version directly should be functionally equivalent however.